03.image_classification/README.cn.md

```diff
@@ -403,66 +378,76 @@ cost_ploter = Ploter(train_title, test_title)
 step = 0
 def event_handler_plot(event):
     global step
-    if isinstance(event, paddle.event.EndIteration):
+    if isinstance(event, fluid.EndStepEvent):
         if step % 1 == 0:
```

Comment:
we should change `step % 1 == 0` to something more meaningful, say `step % 10`

Reply:
How about we remove this line? Looks like we should draw it no matter what.